restream_job.py

```diff
@@ -1,9 +1,23 @@
+from os import environ
+from requests import exceptions
 from api_config import api
 
+MAX_RETRIES = os.environ.get('RESTREAM_MAX_RETRIES', 10)
+
 if __name__ == '__main__':
-    print "Restreaming alooma queue."
-    try:
-      api.start_restream()
-      print "Successfully restreamed alooma queue."
-    except Exception as e:
-      print e
+  print "Restreaming alooma queue."
+  retries = 0
+
+  while True:
```

Comment:
I'm always nervous about messing up `while True` and ending up with an infinite loop. What do you think about this approach instead? http://xion.io/post/code/python-retry-idiom.html

Reply:
Yes, done. I left out the second 'else' statement because I don't think we need to actually raise the error.